libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -146,6 +147,13 @@ const AP_Param::GroupInfo AC_Fence::var_info[] = {
     // @User: Standard
     AP_GROUPINFO_FRAME("OPTIONS", 11, AC_Fence, _options, static_cast<uint16_t>(AC_FENCE_OPTIONS_DEFAULT), AP_PARAM_FRAME_PLANE | AP_PARAM_FRAME_COPTER | AP_PARAM_FRAME_TRICOPTER | AP_PARAM_FRAME_HELI),
 
+    // @Param: ALT_FRAME
+    // @DisplayName: Altitude frame
+    // @Description: The altitude reference frame for all altitude fences. Can be AMSL, home-relative, origin relative or AGL. Applies to both FENCE_ALT_MAX and FENCE_ALT_MIN.
```

Comment:
I was thinking one for min, one for max.

For those users that don't want to come within n metres of the surface but do have an actual AMSL flight level limit.

Reply:
Fixed in https://github.com/ArduPilot/ardupilot/pull/31619